packages/zoneify/src/util/logger.test.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-console */
```

Comment:
What do you think about adding something like 👇 in package.json rules, sometimes in the future?

```
"eslint-disable no-console": [
        "error",
        {
          "devDependencies": [
            "**/*.test.js",
          ]
        }
      ]
```

Reply:
Yeah not against it. Given I think there is only one place current I'm just using a one off exception.